pkgs/development/python-modules/libnl/default.nix

```diff
@@ -0,0 +1,25 @@
+{ buildPythonPackage, isPy3k, libnl, swig, file }:
+buildPythonPackage {
+  name = "libnl";
+  inherit (libnl) version src patches meta;
+
+  disabled = isPy3k;
+
+  nativeBuildInputs = libnl.nativeBuildInputs ++ [  swig file ];
+
+  propagatedBuildInputs = [ libnl ];
+
+  # needed to generate setup.py from setup.py.in
+  configurePhase = ''
+    runHook preConfigure
+    ./configure
+    runHook postConfigure
+  '';
+
+  postConfigure=''
+    cd python
```

Comment:
Maybe we should have a python output in `libnl` instead? What python versions does libnl support at the moment?